### ISSUE TITLE
Installer should report information to whichever "console" is currently active

### ISSUE THREAD

**Issue by konobi** [joyent/sdc#24]
Currently it looks like it always goes to "serial" even if i've manually set console=text. It makes it look like absolutely nothing is going on, especially in error cases.

**Comment by nshalman** [comment]
I think nowadays grub should be rendering on both the VGA and the serial port and with OS-3634 and OS-3635 it should be trying pretty hard to find a working serial port without compromising rendering performance in the absence of one.

**Comment by trentm** [comment]
@nshalman I got an *email* about a comment you made on this ticket, but no longer. Did you delete it?

**Comment by nshalman** [comment]
Yes. After rereading the follow up comment I realized that it didn't quite add anything useful to the conversation.
